sdk/src/main/java/com/akkaserverless/javasdk/action/ActionContext.java

```diff
@@ -41,4 +42,14 @@
    * emitted from an entity.
    */
   Optional<String> eventSubject();
+
+  /**
+   * Try to deserialize the protobuf any message to the given class annotated
+   *
+   * @param jsonClass The class to deserialize to.
+   * @param any A protobuf Any message with the type `json.akkaserverless.com/JsonValue`.
+   * @param <T> The type of the deserialized message
+   * @return The deserialized message
+   */
+  <T> T deserializeJson(Class<T> jsonClass, Any any);
```

Comment:
since this is more like a utility function we don't have to provide it via the context, right? Place it in some helper class? JsonConverter? JsonSupport? JsonDecoder?

Reply:
It currently needs access to the `AnySupport` instance that comes from the service to do the actual json deserialization. But perhaps it could be extracted from there, I'll look into that.

Reply:
ah, I missed that part. I think AnySupport is per component so makes sense to add it to the context (or the Action base class)

Reply:
I actually think the any support state is not needed, I'll make it a utility helper instead.

Reply:
Done, now in JsonSupport.encode/decode